package.json

```diff
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "@rebass/mdx": "^1.0.0-1",
-    "styled-components": "^3.3.3"
+    "styled-components": "^4.0.0"
```

Comment:
I'll need to make an upstream change to `x0` before bumping `styled-components` version so we might want to skip this update as part of this PR. It's on my todo list today to get this fixed.

Reply:
OK, perfect, ping back here when you’re done!

Reply:
Or we can delay, whatever works?

Reply:
If you want to keep this pinned at 3 you can merge this and I can PR in this update separately (I don't want to be a blocker, currently on the road with spotty internet access).

Reply:
Downgraded! I believe this is ready now, and could be batched up with the docs changes as a new release!